arc/cli/commands/diff.py

```
@@ -117,271 +80,394 @@
-    )
-    
-    console.print(table)
-    console.print()
-    
-    # Scenario-level changes
-    if scenario_differences['improvement_count'] > 0 or scenario_differences['regression_count'] > 0:
-        console.print_header("Scenario-Level Changes")
+async def _execute_config(
+    orchestrator: ModalOrchestrator,
+    config: AgentConfig,
+    scenarios: List[Dict[str, Any]],
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)